browser-tests/functional-tests/core-back-errors.spec.js

```diff
@@ -0,0 +1,29 @@
+const { test, expect } = require("@playwright/test");
+const { getAuthoriseUrlForJourney } = require("./helpers");
+
+test.describe("Error tests", () => {
+  test("Handles an unexpected error from core-back", async ({ page }) => {
+    // Start a session
+    await page.goto(getAuthoriseUrlForJourney("testError"));
+
+    // Go to the error
```

Comment:
This does test the right thing, but it took me a little while to work out how.
It might fit with the other tests a bit more if this also just submitted the initial form and core back returned an error, rather than using the more unusual "submit and action via a page request" route.